OWNERS_ALIASES

```diff
@@ -27,9 +27,8 @@ aliases:
 
   # folks who can review and LGTM any PRs in the repo
   cluster-api-reviewers:
-  - CecileRobertMichon
-  - vincepri
```

Comment:
@vincepri removed you and I from reviewers since we're both maintainers, I don't think there is value in having us listed twice

Reply:
IIRC reviewers get randomly assigned to PR reviews, but that list usually doesn't include approvers/maintainers as well.

Reply:
I think you're right. The default logic (which should be active in our repo, I assume we're using blunderbuss) is:
* pick two reviewers
* if there aren't enough reviewers pick approvers

So I think with the config on this PR always Joel and I will be assigned as reviewers. But I might be wrong

Some links:
https://github.com/kubernetes/test-infra/blob/1a550c281187f9eb11bdbcb4272669f213f048dd/config/prow/plugins.yaml#L202-L204
https://github.com/kubernetes/test-infra/blob/c273d05a692dd96bd15344528aa90cb9978d8bb5/prow/plugins/config.go#L143-L147
https://github.com/kubernetes/test-infra/blob/895e1d55e1593b9f550aff7c1c25ae6dcd949b96/prow/plugins/blunderbuss/blunderbuss.go#L233-L239

Reply:
Does that really matter to you @vincepri ? I don't know about you but I'm going to go through the PR queue and review PRs regardless of whether the bot assigned me or not... If do we care about assigning approvers as reviewers then it should be the full approver list, not just us 2 IMO

Reply:
That's a good point. I personally also work based on Github notifications and a watch on the whole repo and not based on Prow assignments.

Reply:
I've always wondered why @vincepri gets assigned reviews on CAPA PRs even though I took him out. Where would I put `ExcludeApprovers`  in the config?

Reply:
As far as I know the blunderbuss plugin is configured globally in prow: (in contrast to approve / blockade)
* https://github.com/kubernetes/test-infra/blob/c273d05a692dd96bd15344528aa90cb9978d8bb5/prow/plugins/config.go#L65-L67
* https://github.com/kubernetes/test-infra/blob/1a550c281187f9eb11bdbcb4272669f213f048dd/config/prow/plugins.yaml#L202-L204
So I think this needs a change in Prow, to be able to also configure this on an org or repo level.

Reply:
> Does that really matter to you @vincepri ?

Nope, mostly wanted to make sure we were all aware